grails-app/views/purchaseOrderWorkflow/purchaseOrder/showOrderItems.gsp

```diff
@@ -58,7 +63,7 @@
                             <!-- data is dynamically loaded -->
                         </tbody>
                         <tfoot>
-                            <g:if test="${order?.status < org.pih.warehouse.order.OrderStatus.PLACED}">
+                            <g:if test="${isApprover.toBoolean()}">
```

Comment:
I still don't understand why the logic here ignores the order status. Can you explain that?

Reply:
"allow add line to PO at any point (with correct permissions)"

Reply:
And in scenario 3 in the ticket, there is specified that even if the order is placed user can add a new line to it. Should I add here:
`order?.status < org.pih.warehouse.order.OrderStatus.RECEIVED`? Or `<=PLACED` or any other status? (but still the bullet point from the planning meeting sais "at any point")

Reply:
No, I hadn't read the requirements all the way through. I just thought it was weird. I'll add a comment to the ticket to reflect what you mentioned in your last comment.